### PR TITLE
GS-90: Reorganise page layout

### PR DESCRIPTION
**Pull request by coldrunKacper**
- Reorganising Pivot Report page layout.
- Changing no-cache message and adding a link to Pivot Report Config page (created with GS-91).

No-cache message:
gs-90-no-cache

Pivot Report page layout after changes:
gs-90-pivot-report-view3

### CODE REVIEW

**Comment by jamienovick** [comment]
@MiyaNoctem I don't think the export buttons should be by the configurations.

Can we put these above the report on same line as chart type but right aligned?

File: templates/CRM/PivotReport/Page/ReportConfig.tpl [hunk 1]
```diff
@@ -8,29 +8,29 @@
...
+          <input type="button" class="report-config-save-btn btn btn-primary hidden" value="{ts}Save Report{/ts}">
+        </div>
+        <div class="form-item">
+          <input type="button" class="report-config-save-new-btn btn btn-primary" value="{ts}Save As New{/ts}">
+        </div>
+        <div class="form-item">
+          <input type="button" class="report-config-delete-btn btn btn-danger hidden" value="{ts}Delete{/ts}">
+        </div>
+      {/if}
+      <div class="form-item right-align">
```
Comment (MiyaNoctem) [inline templates/CRM/PivotReport/Page/ReportConfig.tpl]: As per @jamienovick 's comment, we should probably move these two divs with the export buttons into #pivot-report-type div.

**Comment by jamienovick** [comment]
much better thanks!

**Comment by jamienovick** [comment]
Oh @coldrunKacper can we remove the print icon from that page as that does not work!

**Comment by coldrunKacper** [comment]
@jamienovick Print icon removed.

**Comment by jamienovick** [comment]
ace thanks - client asked for this specifically.

**Pull request merged**